app/views/denounces/index.html.erb

```diff
@@ -0,0 +1,118 @@
+<div class="row justify-content-center mt-5">
+  <div class="col-xs-12 col-lg-10">
+    <div class="card border-primary">
+      <div class="card-body">
+        <div class="row d-flex align-items-center">
+          <div class="col-xs-10 col-lg-6">
+            <form class="row d-flex justify-content-xs-center">
+              <div class="col-xs-3 col-sm-6">
+                <input class="form-control" type="search" placeholder="Search" aria-label="Search">
+              </div>
+              <div class="col-xs-6 col-sm-5">
+                <button class="btn btn-secondary my-xs-2" type="submit"><i class="bi bi-search mt-xs-2">  Search</i></button>
+              </div>
+            </form>
+          </div>
+          <div class="col-xs-12 col-lg-6 d-flex justify-content-xs-center justify-content-lg-end">
+            <%= link_to new_denounce_path, class: "btn btn-outline-primary my-2" do %>
+              <i class="bi bi-plus-circle">  Add new Denounce</i>
+            <% end %>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="col-xs-12 col-md-10 my-4">
+    <h2 class="text-center">My Denounces</h2>
+    <hr class="my-4">
+    <div class="row">
+      <div class="accordion my-5" id="accordionDenounces">
+        <% @denounces.each do |denounce| %>
+          <div class="col-xs-12">
+            <div class="accordion-item" >
+              <div class="accordion-header" id=<%= "denouncesHeading-#{denounce.id}"%>>
+                <div class="row my-1 justify-content-between align-items-lg-center">
+                  <div class="col-xs-12 col-lg-3 d-flex justify-content-center">
+                    Date: <%= Time.at(denounce.created_at.to_i).to_formatted_s(:long_ordinal) %>
```

Comment:
This time format could be in a helper instead

Reply:
Got it we are going to create a new branch and done all the necessary refactor there. Because We make a mistake and now we have dependency of tasks. So we want to merge this PR and then make smaller Pr's with changes or refactorings. Because this is working.

Reply:
I'm going to mark as resolved for now. Feel free to unresolved if you disagree with the decision made.